packages/desktop-client/src/components/accounts/Balance.tsx

```diff
@@ -126,34 +148,58 @@ function FilteredBalance({ filteredAmount }) {
   );
 }
 
-function MoreBalances({ balanceQuery }) {
+function MoreBalances({
+  balanceQuery,
+}: {
+  balanceQuery: ComponentProps<typeof ReconcilingMessage>['balanceQuery'];
+}) {
   const { t } = useTranslation();
 
-  const cleared = useSheetValue({
-    name: balanceQuery.name + '-cleared',
+  type SelectedBalanceClearedName = `balance-query-${string}-cleared`;
+  const cleared = useSheetValue<'balance', SelectedBalanceClearedName>({
+    name: (balanceQuery.name + '-cleared') as SelectedBalanceClearedName,
```

Comment:
This works. 

There must be a better way to write this, but I can't figure out how.